compiler/src/lib.rs

```diff
@@ -331,6 +333,12 @@ fn compile<T: FieldElement, Q: QueryCallback<T>>(
 
 #[allow(clippy::print_stdout)]
 pub fn inputs_to_query_callback<T: FieldElement>(inputs: Vec<T>) -> impl QueryCallback<T> {
+    // TODO: This is a copy of riscv::bootloader::default_input(), so we don't have to
+    // introduce a dependency to the `riscv` crate.
+    // In the future, we should expose bootloader inputs.
+    let mut bootloader_inputs = [T::zero(); 38];
+    bootloader_inputs[36] = T::from(51u64);
```

Comment:
All these hardcoded values sound quite fragile and need to be consistent. Can we make them const somewhere and just use that?

Reply:
Actually, I just set the bootloader inputs to the empty array, so it always fails if they are accessed during witness generation. In the future, this will be fixed by passing them to create the callback, so then they can come from the `riscv` crate!